webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/AppConfigAuth.scala

```diff
@@ -158,10 +166,20 @@ class AppConfigAuth extends ApplicationContextAware {
       } catch {
         case ex: ConfigException.Missing => // does nothing - the beauty of imperative prog :(
       }
-      val properties: Option[(String, AuthBackendProviderProperties)] = x.name match {
+      val restTokenFeatureSwitch: FeatureSwitch                                   = {
+        // special feature switch depends on global one.
+        restTokenGlobalFeatureSwitch match {
+          case FeatureSwitch.Disabled => FeatureSwitch.Disabled
+          case FeatureSwitch.Enabled  =>
+            try {
+              FeatureSwitch.parse(config.getString(s"rudder.auth.${x.name}.userRestToken")).getOrElse(defaultEnableRestToken)
+            } catch { case ex: ConfigException.Missing => defaultEnableRestToken }
```

Comment:
same here: info application log in the case where the value is different from the global one, debug level when it's the same